agent-svcat/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM quay.io/feedhenry/jenkins-agent-base-centos7:latest
+
+MAINTAINER Vitalii Chepeliuk <vchepeli@redhat.com>
+
+USER root
+
+ARG SVCAT_VER=0.1.21
+ARG SVCAT_BIN=svcat
+ARG SVCAT_URL=https://download.svcat.sh/cli/v${SVCAT_VER}/linux/amd64/${SVCAT_BIN}
+ARG MBCLI_VER=1.0.0
+ARG MBCLI_BIN=mobile
+ARG MBCLI_GZIP=mobile-cli_${MBCLI_VER}_linux_amd64.tar.gz
+ARG MBCLI_URL=https://github.com/aerogear/mobile-cli/releases/download/${MBCLI_VER}/${MBCLI_GZIP}
+ENV SA_DIR /var/run/secrets/kubernetes.io/serviceaccount
+ENV URL kubernetes.default.svc.cluster.local
+
+RUN yum -y update && \
+    yum -y install \
+        curl \
+        unzip \
+        wget \
+        zip && \
+    yum clean all
+
+RUN cd /tmp && curl -s -O -J -L ${SVCAT_URL} && chmod +x ${SVCAT_BIN} && mv ${SVCAT_BIN} /usr/local/bin/${SVCAT_BIN}
+RUN cd /tmp && curl -s -O -J -L ${MBCLI_URL} && tar -xf ${MBCLI_GZIP} && mv ${MBCLI_BIN} /usr/local/bin/${MBCLI_BIN}
```

Comment:
Has this version been pushed to quay.io/feedhenry/jenkins-agent-base-centos7:latest?

Reply:
nope. Does not it pushed automatically? Or how I can push that?

Reply:
@mikenairn Built from Dockerfile here: https://quay.io/repository/feedhenry/jenkins-agent-svcat/build/f859a471-f509-44cf-a695-1e5108530fb2